neo4j/src/main/java/org/apache/metamodel/neo4j/Neo4jDataContext.java

```diff
@@ -52,23 +52,24 @@
  * DataContext implementation for Neo4j
  */
 public class Neo4jDataContext extends QueryPostprocessDataContext implements DataContext, DocumentSourceProvider {
-
-    public static final Logger logger = LoggerFactory.getLogger(Neo4jDataContext.class);
-
     public static final String SCHEMA_NAME = "neo4j";
-
     public static final int DEFAULT_PORT = 7474;
-
-    public static final String RELATIONSHIP_PREFIX = "rel_";
-
-    public static final String RELATIONSHIP_COLUMN_SEPARATOR = "#";
+    public static final String NEO4J_KEY_METADATA = "metadata";
```

Comment:
Can you make these newly added constants (all starting with "NEO4J_") package protected? They are all only used for internal logic within this package here and I don't think we want to expose them by default as part of the api.